code/game/turfs/simulated/floor/misc_floor.dm

```diff
@@ -126,6 +126,38 @@
 	burnt_states = list("noslip-scorched1","noslip-scorched2")
 	slowdown = -0.3
 
+/turf/open/floor/noslip/standard
+	name = "high-traction floor"
+	icon_state = "noslip_standard"
+	floor_tile = /obj/item/stack/tile/noslip/standard
+	broken_states = list("noslip-damaged1_standard","noslip-damaged2_standard","noslip-damaged3_standard")
+	burnt_states = list("noslip-scorched1_standard","noslip-scorched2_standard")
+	slowdown = -0.3
```

Comment:
None of these need this line as they all inherit it from `/turf/open/floor/noslip`

Reply:
that's true, lemme fix it real quick